src/doc/src/reference/environment-variables.md

```diff
@@ -353,6 +353,8 @@ let out_dir = env::var("OUT_DIR").unwrap();
     * `CARGO_CFG_TARGET_POINTER_WIDTH=64` --- The CPU [pointer width].
     * `CARGO_CFG_TARGET_ENDIAN=little` --- The CPU [target endianness].
     * `CARGO_CFG_TARGET_FEATURE=mmx,sse` --- List of CPU [target features] enabled.
+  > Note that different [target triple][Target Triple] has a different set of `cfg` values,
```

Comment:
Thanks, just a minor tweak of the wording:

```suggestion
  > Note that different [target triples][Target Triple] have different sets of `cfg` values,
```

Reply:
Updated. Thanks!